services/worker-manager/src/providers/azure.js

```diff
@@ -123,12 +145,12 @@ class AzureProvider extends Provider {
       toSpawn -= cfg.capacityPerInstance;
     }
 
+    // Create "empty" workers to provision in provisionResources loop
     await Promise.all(cfgs.map(async cfg => {
       // This must be unique to currently existing instances and match [a-z]([-a-z0-9]*[a-z0-9])?
       // The lost entropy from downcasing, etc should be ok due to the fact that
       // only running instances need not be identical. We do not use this name to identify
       // workers in taskcluster.
```

Comment:
Is this comment still true?  It seems this value is now used for workerId..

Reply:
Good catch, that second part is not true. According to [this](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) VM names can be up to 64 characters long, so I've bumped that up and added another nicerId() for entropy.

Reply:
It's true, but workerId is limited to 38 characters -- https://community-tc.services.mozilla.com/schemas/worker-manager/v1/worker-full.json